crates/icegate-jobmanager/Makefile

```
@@ -0,0 +1,16 @@
+.PHONY: examples-infra-up examples-infra-down tests
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)